src/metabase/lib/stage.cljc

```diff
@@ -181,35 +217,57 @@
 ;;; PLUS
 ;;;
 ;;; 3. Columns added by joins at this stage
-(defmethod lib.metadata.calculation/default-columns-method ::stage
-  [query stage-number _stage unique-name-fn]
+(mu/defn ^:private previous-stage-or-source-visible-columns :- lib.metadata.calculation/ColumnsWithUniqueAliases
```

Comment:
This is also existing code, it was in a different function and I just split it out